cmd/server/main.go

```diff
@@ -27,7 +29,11 @@ func addAndroidVariant(labels map[string]string, client *upsClient, name string)
 		payload := &androidVariant{
 			ProjectNumber: labels["projectNumber"],
 			GoogleKey:     val,
-			variant: variant{Name:name},
+			variant: variant{
+				Name:       name,
+				VariantID:  uuid.NewV4().String(),
```

Comment:
doesn't the variant id come from the push server?

Reply:
yes - push server can create them too

but for "reading" we need the variandID / secret - but if we dont post them, that means different struct ... not sure that is nice ..